test/Test/Universum/StringProps.hs

```diff
@@ -1,3 +1,8 @@
+-- Some tests in this module have a caveat: they depend on compiler
+-- optimizations and will likely fail if compiled with -O0.
+-- That's because they test are rewrite rules, and these rules
```

Comment:
Something seems to be amiss in this sentence

Reply:
Looks weird, I don't remember what exactly I wanted to write. Anyway, pushed a fix with sane wording.